app/views/contacts/new.html.haml

```diff
@@ -1,7 +1,12 @@
 - if @contact.restroom_name.present?
-  %h1 Request Edit for #{@contact.restroom_name} Restroom
+  - titlestring = t('.request-edit-for')
```

Comment:
You can actually interpolate this. `t('request-edit-for', restroom_name: @contact.restroom_name)` and then define the translation like this:
`request-edit-for: 'Request Edit for %{restroom_name} Restroom'`

Reply:
I18n, takes the param restroom_name and shoves it into the %{restroom_name} param